amplience-automation/automation-files/schema/schemas/product-pdp-schema.json

```diff
@@ -0,0 +1,78 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://sfcc.com/site/product-pdp",
+
+    "title": "Product PDP",
+    "description": "Associates an SFCC product with Amplience content that should appear on its PDP.",
+
+    "allOf": [
+        {
+            "$ref": "http://bigcontent.io/cms/schema/v1/core#/definitions/content"
+        }
+    ],
+
+    "trait:filterable": {
+        "filterBy": [
+            {
+                "paths": [
+                    "/active"
+                ]
+            }
+        ]
+    },
+    
+    "type": "object",
+    "properties": {
+        "active": {
+            "title": "Active",
+            "description": "",
+            "type": "boolean",
+            "default": true
+        },
+        "_meta": {
+            "title": "Associated Product",
+            "properties": {
+                "deliveryKey": {
+                    "title": "Product ID",
+                    "description": "(format pdp/content/<id>) When the Product ID matches, this item's content is shown on the PDP.",
```

Comment:
Update this to say that it is required to show vis (and generally work)?

Reply:
Resolved